### PR TITLE
Latest Articles section in Services site and a small fix for CTA Banner

### PR DESCRIPTION
**Pull request by sc-nelikostadinova**

### CODE REVIEW

File: src/sxastarter/src/components/PageContent/ArticleList.tsx [hunk 1]
```diff
@@ -111,11 +110,6 @@ export const ThreeColumn = (props: ArticleListComponentProps): JSX.Element => {
       id={id ? id : undefined}
     >
       <div className="container">
-        <div className="row">
-          <div className="col">
-            <div className="title display-6">{t('More News') || 'More News'}</div>
```
Comment (adoprog) [inline src/sxastarter/src/components/PageContent/ArticleList.tsx:116]: Is this intended? Don't we need it on Financial?

File: src/sxastarter/src/components/PageContent/CtaBanner.tsx [hunk 2]
```diff
@@ -57,10 +56,6 @@ export const Default = (props: CtaBannerProps): JSX.Element => {
                 <div className="fs-5">
                   <RichText field={props.fields.Text} className="text-content" />
 
-                  <div className="row mt-2">
-                    <Placeholder name="promo-cta" rendering={props.rendering} />
```
Comment (adoprog) [inline src/sxastarter/src/components/PageContent/CtaBanner.tsx:61]: Also not needed anymore?

**Review by adoprog — APPROVED** [review]

File: src/sxastarter/src/components/PageContent/CtaBanner.tsx (hunk 2 above)
Comment (sc-nelikostadinova) [reply src/sxastarter/src/components/PageContent/CtaBanner.tsx:61]: This was forgotten, it was never used.

File: src/sxastarter/src/components/PageContent/ArticleList.tsx (hunk 1 above)
Comment (sc-nelikostadinova) [reply src/sxastarter/src/components/PageContent/ArticleList.tsx:116]: We are not using this variant anywhere in Financial site

**Pull request merged**